js/src/personal/riyuan/components/experience/tabitem.tsx

```diff
@@ -0,0 +1,112 @@
+import { List, Tabs, Text } from '@mantine/core'
+import styles from '@/personal/riyuan/Riyuan.module.css'
+
+interface TabPanelProps {
+  value: string
+  title: string
+  items: string[]
+}
+
+const TabPanel = ({ value, title, items }: TabPanelProps) => (
```

Comment:
One component per file is generally preferred until you get a better hang of things.